ui/admin/tests/acceptance/host-catalogs/host-sets/hosts-test.js

```diff
@@ -51,6 +51,12 @@ module('Acceptance | host-catalogs | host-sets | hosts', function (hooks) {
     newHost: null,
   };
 
+  const HOST_ACTIONS_SELECTOR =
```

Comment:
I notice we add within the view code a `data-test-` attribute, but I do not see any of the tests using it. Should we delete the `data-test` attribute, or should we add another one for the dropdown and rely of `data-test` attributes rather than long selectors? 

Reply:
@calcaide initially I was having some problems grabbing the element in the test environment.  I will remove this.

Reply:
@dvonanderson 
- Can you describe what problems are you having? We are using `data-test-*` selectors without problems within other tests.
- `but it seemed that some used date-test selectors and some don't.` what do you mean by this?
- Feel free to bring any topic to tech time, tech topics are always welcome in such space. That said, the `data-test-*` selectors is something we use to facilitate our lives (and getting some performance win for free) when have to select elements where the pattern to select is too long or too cumbersome. I.e: `tbody .hds-table__tr:nth-child(1) .hds-dropdown-list-item button` is quite long and will be easier for the developer and for the machine that process this test to use a data-test-* selector, i.e: `data-test-host-set-hosts-remove`. I think we should apply common sense, if the selector is  short ie: `main#main-wrapper` makes sense to use such selector but in other cases `data-test` are preferable.
- What do you mean by: `ersonally I think we should adopt the same patterns that cloud-ui uses.`? We actually copy the `data-test-*` approach from `cloud-ui`

Reply:
@calcaide 
- So originally I think what was happening with the test was. I had `'[data-test-host-set-host-dropdown-remove-host] button'` as the selector for the `dropdown.interactive`.  I just tested it again without the button and it seemed to work.  Maybe I just needed to restart my test and never did. 
- I used a pattern from the auth-methods create test  to set up the selectors for this test.  I do agree its a bit long and cubersome and `data-test`selectors are much more readable.
- Lastly the comment about how `cloud-ui` use selectors refers more to the using the `selectors.js` to house all of your test-support.  

Incoming commit to change it to a `data-test` selector

Reply:
@dvonanderson 

- Yeah not sure about that, perhaps was the environment. Or maybe was you need to open the dropdown first for the remove button to appear? `Hds::Dropdown` does not render the options of the dropdown until is not open, therefor the `[data-test-host-set-host-dropdown-remove-host] button` will not exist on the DOM unless you click the dropdown first to open. NOT saying this is the cause, but can be a possibility. You can check this opening the secreen with the browser dev tools, using the element inspector you will see no remove button unless you open the dropdown first.
- Not mandatory at all, just a suggestion: since you mention [the auth-methods create test](https://github.com/hashicorp/boundary-ui/blob/ade1601d8d533d620feddba6469b69df431a2315/ui/admin/tests/acceptance/auth-methods/create-test.js#L22) in there, the selectors that are used more than once are defined as constants at the beginning of the file and then call the constant during the test file.
- Indeed `data-test` attributes need more definition from our part, feel free to bring that topic to tech time, it needs some discussion: when to use `data-test` selectors vs when to not use them, should we use contants in our test when repeting the same selectors over and over? should we have a file containing all the selectors as `cloud-ui` does with the mentioned `selectors.js`?

Reply:
@calcaide Awesome thank you for the information!!